Core/Graphics/Source/GraphicsImpl.h

```diff
@@ -73,6 +74,7 @@ namespace Babylon
         template<typename WindowT>
         WindowT GetNativeWindow();
         void SetNativeWindow(const GraphicsConfiguration& config);
+        void SetNativeContext(const ContextConfiguration& config);
```

Comment:
Should these both just be called `UpdateConfiguration` to be more symmetrical with creation?

Reply:
Similar for the parent functions (UpdateWindow and UpdateContext) in Graphics.h